ruby/vampires.rb

```diff
@@ -0,0 +1,50 @@
+puts "How many employees will be processed?"
+x = gets.to_i
+x.times do 
+
+	puts "What is your name?"
+	user_name = gets.chomp
+
+	puts "How old are you?"
+	user_age = gets.to_i
+
+	puts "what year were you born?"
+	user_birthyear = gets.to_i
```

Comment:
To be consistent with the Ruby style, make sure the separate all words with `_`, `user_birth_year`